client/src/recoil/atoms/user.ts

```diff
@@ -1,7 +1,19 @@
-import { atom } from 'recoil';
+import { atom, selector } from 'recoil';
 import { User } from 'src/types';
 
 export const userState = atom<User | null>({
   key: 'User',
   default: null,
 });
+
+export const getPrimaryRegionCode = selector({
+  key: 'getPrimaryRegionCode',
+  get: ({ get }: { get: any }) => {
+    const user = get(userState);
+    if (!user?.userRegions) return 0;
+    const primaryRegion = user?.userRegions.find(
+      (region: any) => region.isPrimary === true
+    );
+    return primaryRegion?.region.code || 0;
+  },
```

Comment:
get이랑 region 타입에 any는 지워주셔도 될것 같아요
```suggestion
  get: ({ get }) => {
    const user = get(userState);
    if (!user?.userRegions) return 0;
    const primaryRegion = user?.userRegions.find(
      (region) => region.isPrimary === true
    );
    return primaryRegion?.region.code || 0;
  },
```

Reply:
넵!